_config.yml

```diff
@@ -739,15 +739,20 @@ defaults:
       toc: "sync-service"
       support: true
       versions:
+        - 3.4
         - 3.3
         - 3.2
         - 3.1
         - 3.0
   - scope:
       path: "sync-service/latest"
     values:
-      version: 3.3
+      version: 3.4
       latest: true
+  - scope:
+      path: "sync-service/3.3"
```

Comment:
why path: "sync-service/3.3" shouldn't be "sync-service/3.4" ?

Reply:
Great questions! This config file is used to build the Docs site, and sets the versions shown in the content pages. I've added an explanation of what's going on in the PR description. This should address all your questions so far.

So, in this file, the key changes are:
- moving v3.3 content from "latest" to "sync-service/3.3"
- setting v3.4 as the "latest" version (see lines 746 - 750)

HTH